docs/ja/jump-start-plasma.md

```diff
@@ -1,16 +1,16 @@
 ---
-id: jump-start-plasma
-title: plasma Jump Start
-sidebar_label: plasma Jump Start
+id: jump-start-basechain
+title: Loom Basechain Jump Start
+sidebar_label: Loom Basechain Jump Start
 ---
 
-Follow the instructions below to launch a plasma node from the jump start
+Follow the instructions below to launch a Loom Basechain node from the jump start
 
-## Download stable version of plasmachain
+## Download stable version of Loom Basechain
 
 ```bash
-wget https://private.delegatecall.com/loom/linux/stable/plasmachain
-chmod +x plasmachain
+wget https://downloads.loomx.io/loom/linux/latest/basechain
+chmod +x basechain
```

Comment:
@andreipope Don't forget the download links below too. The filename prefix will be `plasma-` should be `basechain-`.